examples/irsa-xr.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: aws.platformref.upbound.io/v1alpha1
+kind: XIRSA
```

Comment:
a little late here, but is there any README updates or docs of some sort that can explain this new API and how to use it? :)